icicle/include/icicle/msm.h

```diff
@@ -67,9 +67,17 @@ namespace icicle {
       1,       // batch_size
       true,    // are_points_shared_in_batch
       false,   // are_scalars_on_device
-      false,   // are_scalars_montgomery_form
-      false,   // are_points_on_device
-      false,   // are_points_montgomery_form
+#ifdef BARRET
+      false, // are_scalars_montgomery_form
+#else
+      true, // are_scalars_montgomery_form
```

Comment:
this also can explain the performance since we have to pay conversion per scalar

Reply:
when testing I wasn't using the default config, I use the configuration that will not require any conversions, in both barret and mont